.devcontainer/devcontainer.json

```diff
@@ -7,24 +7,18 @@
 //      container, but if you use, for example, the "capAdd" JSON field then it will NOT be picked up by the script
 //      without changes
 {
-	"dockerFile": "Dockerfile",
-	"build": {
-		"args": {
-			"WORKSPACE": "${containerWorkspaceFolder}"
-		}
-	},
+	"name": "Sailbot Workspace",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "sailbot-workspace",
+	"workspaceFolder": "/workspaces/sailbot_workspace",
 	"remoteUser": "ros",
-	// Do not add comments to any lines within the runArgs array - it breaks the start_container.sh script
-	"runArgs": [
-		"--cap-add=SYS_PTRACE",
-		"--cap-add=NET_ADMIN",
-		"--network=host",
-		"--security-opt=seccomp:unconfined",
-		"--security-opt=apparmor:unconfined"
-	],
```

Comment:
@hhenry01 what is `--network=host` for? I added all the run arguments to the docker compose file except for this, cause I need the network to be the same as the mongodb database: `service:mongodb`

Resources:
- host networking: https://docs.docker.com/network/host/
- compose networking: https://docs.docker.com/compose/compose-file/compose-file-v3/#network_mode

Reply:
If we're not going to use can/vcan with the dev container I think we can hardcode these in the deployment script rather than parsing it from somewhere else